### ISSUE TITLE
[Request] Make popover accessible to use it for custom actions

### ISSUE THREAD

**Issue by TheNaderio** [fullcalendar/fullcalendar#4643]
This popover from the "more" function is great but i cant find any documented way to trigger this popover within custom functions.

![image](https://user-images.githubusercontent.com/25088164/56679500-7b049880-66c5-11e9-88be-8dd80d572fab.png)

I want to use this popover to render an event preview inside of it. 
In my special case i want to fill it with eventdata which eventually will come from an ajax request.

As result I want to create something like this:

![image](https://user-images.githubusercontent.com/25088164/56679464-6c1de600-66c5-11e9-8df5-6fc0bb0297a9.png)

I think this would be a great feature, without it I now have to use an other third party plugin (or create something on my own) to show the popover and position it related to the clicked Event or day.

Thanks  😃👍

**Comment by acerix** [comment]
It's preferred to keep things like this outside the calendar.

You can attach a popup in `eventRender` using the event's element so it displays in the correct position:

https://fullcalendar.io/docs/eventRender

**Comment by TheNaderio** [comment]
Yes, but it is not possible to use the popover, that is already there. I need to add e.g. tooltip.js and call it.
I think it would be very useful if you could call an empty `fc-popover`-Container in a certain position and fill it with data.

**Comment by arshaw** [comment]
I understand it might be useful to trigger the "more" popover programatically.

**Comment by Herz3h** [comment]
Since eventRender does not exist in v5 anymore, and evenContent hook only allows for inner html (anything inside `.fc-event`), I have tried but tooltip never shows. 

Is there a 'higher' hook than evenContent ?

**Comment by eyea** [comment]
> I can see how that would be useful, but it's preferred to keep things like this outside the calendar.
> 
> You can attach a popup in `eventRender` using the event's element so it displays in the correct position:
> 
> https://fullcalendar.io/docs/eventRender

BUT, the eventRender does not exist in v5 anymore.

**Comment by belozer-ti** [comment]
@eyea  you need use `eventContent`

**Comment by eyea** [comment]
> @eyea you need use `eventContent`

OK, thank you :)

**Comment by alexandraseri** [comment]
Actually I'm trying to do something similar while in weekly view - open a popover programmatically for an event, but if I have an event that is more than 1 day - for example, 4 days, I need to know which popover to open, i.e. the one on the first day, or second day, etc.
`eventContent` doesn't provide me with the day on which the event is rendering - so I don't know which popover to open.

**Comment by Xezz22** [comment]
I want to click the event to open the popover , but how can I get the position that the popover  should display
Thank you :)

**Comment by arshaw** [comment]
I'm re-triaging this ticket because I think I misclassified it. There are a number of issues being talked about here:

- @TheNaderio (the original poster) - I'm not sure I want FullCalendar to expose a general lib for rendering popovers. The value-add of FullCalendar doing it would be for positioning features: to ensure the popover is anchored to a specific event or day. However, it's not too difficult to achieve this already. It's easy to obtain the DOM element for a specific event or day via the various callback hooks. You can also grab an day's element anytime by using a querySelector like `td[data-date="2022-12-12")`
- @Herz3h / @eyea / @belozer-ti - The `eventDidMount` [render hook](https://fullcalendar.io/docs/event-render-hooks) will allow you to get the top-level DOM element of the event just like `eventRender` did in v4.
- @alexandraseri - to know which day element was clicked for a multi-day event, [this issue](https://github.com/fullcalendar/fullcalendar/issues/2430) is the one to follow

For the posters that had additional questions, can you please post separate tickets and/or use Stackoverflow?
